### ISSUE TITLE
Serviceapp seems to report some Mpeg2 as Mpeg1

### ISSUE THREAD

**Issue by Ev0-BH** [mx3L/serviceapp#53]
Some iptv channels get reported as Mpeg1 when using serviceapp with exteplayer3, when they are really Mpeg2.

For most boxes this isn't a problem, however Vu+ Zero4K does not support Mpeg1, so even though the video is Mpeg2 (verified using Gstreamer instead), you don't receive the video, only audio when using serviceapp.

**Comment by e2iplayer** [comment]
This was issue in the exteplayer3 not in the serviceapp.
It was fiedx in
https://github.com/e2iplayer/exteplayer3/commit/857e07c208204766fa58dc364bf9e9596e136f31

**Comment by Ev0-BH** [comment]
Hi, yes I noticed SSS fixed it a few days ago in exteplayer3 :)

**Comment by Ev0-BH** [comment]
@e2iplayer 

Do you think it would be possible to report H.264 4K video as H.265 so those boxes that do not support H.264 4K would get video (in the same way as boxes that did not support Mpeg1 did not get video) ?

**Comment by e2iplayer** [comment]
No. This is completely two different things. 
If your STB does not support H.264 4K is most probably due to hardware limitation. This is not couse by wrong video decoder configuration as it was in case MPEG2.

**Comment by Ev0-BH** [comment]
Fair enough, I only thought it might work, because the Zero4K and Duo4K do not support Mpeg1 (hardware limitation), so they did not get video only audio when streams were reported as Mpeg1.

Those same streams now work fine with video and audio as they now report as Mpeg2.

When you try and play H.264 4K video on Ultimo4K for example, you get no video only audio, so thought it might work in a similar way.
